### PR TITLE
[next] Implement "Read More of this Conversation"

### PR DESCRIPTION
**Pull request by alinhle**
## What does this PR do?
- https://www.pivotaltracker.com/story/show/160729882

## How do I test this PR?

- Write comments until max depth levels has been exceeded
- Reload and see a functional `Read More of this Conversation` button

### CODE REVIEW

File: src/core/client/stream/tabs/comments/containers/CommentContainer.tsx [hunk 1]
```diff
@@ -113,13 +122,20 @@ export class CommentContainer extends Component<InnerProps, State> {
     return;
   }
 
+  private handleShowConversation = (e: MouseEvent) => {
```
Comment (okbel) [inline src/core/client/stream/tabs/comments/containers/CommentContainer.tsx:125]: 👍 to MouseEvent

**Comment by okbel** [comment]
1. Adding 6 levels of comments. 
2. Clicking Read More
3. Replying to the comment thread i.e `http://localhost:8080/?commentID=3d73d711-aa92-4bc5-af30-1ca06c36951f` does not work.

Is that expected? Or is it handled in the next threading PR?

**Comment by okbel** [comment]
Also this error shows up `Warning: ReactRelayPaginationContainer: Cannot paginate without pageInfo fields in `ReplyListContainer`. Be sure to fetch hasNextPage (got `null`) and endCursor (got `null`).` But I think that's the one being handled in the next threading PR, right?

**Review by okbel — CHANGES_REQUESTED** [review]
See the comments above.

**Comment by alinhle** [comment]
Yes those are all handled in the next PR.

**Review by okbel — APPROVED** [review]
👍

**Pull request merged**